pkgs/tools/networking/termscp/default.nix

```diff
@@ -49,7 +51,7 @@ rustPlatform.buildRustPackage rec {
   doCheck = false;
 
   meta = with lib; {
-    description = "Terminal tool for file transfer and explorer";
+    description = "🖥 A feature rich terminal UI file transfer and explorer with support for SCP/SFTP/FTP/S3/SMB";
```

Comment:
nothing wrong, just thought the description from project is a little more descriptive  
and when one searches for terminal ui and scp, it would show up  
  
regarding the emoji, I hope it's not a problem but I'll remove it anyway

Reply:
The description should be short and on the point as this is the element a lot of tools are parsing. Everything else can always go into `longDescription`. IMHO, "feature rich", "super fast", "only tool in the world" are often nothing more than marketing blabla.